client/extensions/zoninator/components/settings/zone/index.jsx

```diff
@@ -1,37 +1,93 @@
 /**
  * External dependencies
  */
-import React from 'react';
-import PropTypes from 'prop-types';
+import React, { Component } from 'react';
 import { connect } from 'react-redux';
 import { localize } from 'i18n-calypso';
-import { flowRight, noop } from 'lodash';
+import { flowRight } from 'lodash';
+import { FieldArray, FormSection, reduxForm } from 'redux-form';
 
 /**
  * Internal dependencies
  */
+import PostsList from './posts-list';
+import Button from 'components/button';
+import Card from 'components/card';
+import FormButton from 'components/forms/form-button';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
 import HeaderCake from 'components/header-cake';
+import ReduxFormTextarea from 'components/redux-forms/redux-form-textarea';
+import ReduxFormTextInput from 'components/redux-forms/redux-form-text-input';
+import SectionHeader from 'components/section-header';
 import { getSelectedSiteSlug } from 'state/ui/selectors';
+import { settingsPath } from '../../../app/util';
 
-const Zone = ( { siteSlug, translate } ) => (
-	<div>
-		<HeaderCake backHref={ `/extensions/zoninator/${ siteSlug }` } onClick={ noop }>
-			{ translate( 'Edit zone' ) }
-		</HeaderCake>
-	</div>
-);
-
-Zone.propTypes = {
-	siteSlug: PropTypes.string,
-};
-
-const connectComponent = connect( state => {
-	return {
-		siteSlug: getSelectedSiteSlug( state ),
-	};
+const form = 'extensions.zoninator.editZone';
+
+class Zone extends Component {
+
+	onFormSubmit = ( event ) => {
+		event.preventDefault();
+	}
```

Comment:
This is essentially a no-op. Can we just drop the `onSubmit` from the `form` and remove this?

Reply:
I don't quite get why this is here either. Is it a Redux Form thing or just a leftover bit that can be removed?